src/app.jsx

```diff
@@ -550,6 +565,15 @@ class Application extends React.Component {
 
         cockpit.user().then(user => {
             this.setState({ currentUser: user.name || _("User") });
+            this.getUserLingeringEnabled(user.id);
+            // HACK: https://github.com/systemd/systemd/issues/22244#issuecomment-1210357701
+            cockpit.file(`/var/lib/systemd/linger/${user.name}`).watch((content, tag) => {
```

Comment:
There's no point in querying d-bus if we still have to assume this file location anyway. This watch alone ought to suffice, so let's just drop the dbus property query.